postgresql.html

```diff
@@ -293,7 +289,7 @@
 	/* global RED:false, $:false */
 	RED.nodes.registerType('postgresql', {
 		category: 'storage',
-		color: '#336791',
+		color: '#5b85a7',
```

Comment:
Do you happen to know whether there is any option to make the text white?
This would be the same colours than https://www.postgresql.org/

Reply:
Something like
![image](https://user-images.githubusercontent.com/1008324/144128930-8a6fd362-de32-4ab1-804e-231b984ce893.png)

Reply:
I'm both happy and sad to tell you that no. Text color is forced black and that basically rules out many dark background colors. (I think it makes the UI more unison to not have too many colors changing and so I like it)
 
I realized that you were going for the official colors.  That is why I used this tool to up it two levels of light/tint https://www.color-hex.com/color/336791.